From 3b7c67c9cc68317a7066ab33165d93ffdeb8e8d1 Mon Sep 17 00:00:00 2001
From: Wu Xiaotian <yetist@gmail.com>
Date: Tue, 19 Feb 2019 17:04:25 +0800
Subject: [PATCH 1/2] show full path of caja-extension-dir

---
 configure.ac | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index bf52831..5f061f7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -91,7 +91,11 @@ if test -z "$CAJA_EXTENSION_DIR"; then
     custom_caja_extension_dir=false
 fi
 
-AC_MSG_RESULT([${CAJA_EXTENSION_DIR}])
+if test x$custom_caja_extension_dir = xtrue; then
+    AC_MSG_RESULT([${CAJA_EXTENSION_DIR}])
+else
+    AC_MSG_RESULT([${libdir}/${CAJA_EXTENSION_DIR}])
+fi
 AM_CONDITIONAL([CUSTOM_CAJA_EXTENSION_DIR], [test x$custom_caja_extension_dir = xtrue])
 AC_SUBST(CAJA_EXTENSION_DIR)
 
-- 
2.21.0

