--- a/configure.ac
+++ b/configure.ac
@@ -30,16 +30,16 @@ fi
 PKG_CHECK_MODULES(CAJA, libcaja-extension >= $CAJA_REQUIRED)
 PKG_CHECK_MODULES(GLIB, glib-2.0 >= $GLIB_REQUIRED)
 
-AC_PATH_PROG([PYTHON], [python])
+AC_PATH_PROG([PYTHON], [python2])
 
-AC_PATH_PROG([RST2MAN], [rst2man], [python rst2man.py])
+AC_PATH_PROG([RST2MAN], [rst2man], [python2 rst2man.py])
 AC_SUBST(RST2MAN)
 
 # define module checking macro
 AC_DEFUN([PYTHON_CHECK_MODULE], [
 AC_MSG_CHECKING([for $1])
 
-cat <<EOF | python
+cat <<EOF | python2
 try:
  import $2
 except:
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,11 +6,11 @@ EXTRA_DIST = caja-dropbox.in serializeim
 man_MANS = caja-dropbox.1
 
 caja-dropbox: caja-dropbox.in serializeimages.py
-	python serializeimages.py $(PACKAGE_VERSION) $(datadir)/applications < caja-dropbox.in > caja-dropbox
+	python2 serializeimages.py $(PACKAGE_VERSION) $(datadir)/applications < caja-dropbox.in > caja-dropbox
 	chmod +x caja-dropbox
 
 caja-dropbox.1: caja-dropbox.txt.in caja-dropbox docgen.py
-	python docgen.py $(PACKAGE_VERSION) caja-dropbox.txt.in caja-dropbox.txt
+	python2 docgen.py $(PACKAGE_VERSION) caja-dropbox.txt.in caja-dropbox.txt
 	$(RST2MAN) caja-dropbox.txt > caja-dropbox.1
 
 SUBDIRS = data src
